src/scss/components/person/_person-teasers.scss

```diff
@@ -0,0 +1,47 @@
+@charset "UTF-8";
+
+.ptype-stanford-entity,
+.paragraph--type--stanford-entity {
+
+  .ds-entity--stanford-person {
+    max-width: 400px;
+
+    .su-card {
+      border: none;
+      box-shadow: none;
+
+      .su-person-photo {
+        padding: 0 0 0 7rem;
+      }
+
+      .su-person-photo img,
+      .su-person-photo picture {
+        border-radius: 0;
+      }
+
+      section.su-card__contents {
+        padding: 2rem 3rem 3.2rem 3.2rem;
+        text-align: left;
+
+        h2,
+        h3 {
+          a {
+            @include thub-type-d;
+
+            background: $thub-gradient-lagunita-to-ochre;
+            -webkit-background-clip: text;
+            -webkit-text-fill-color: transparent;
+
+            &:hover,
+            &:focus,
+            &:active {
+              background:  $thub-color-ochre-dark;
```

Comment:
Underline is missing on hover/focus 👀 

<img width="573" alt="image" src="https://github.com/SU-SWS/taiwan_hub_subtheme/assets/34019925/10508640-49b5-40cd-a826-6867d8b0627f">

Reply:
@rebeccahongsf I see an underline on hover on my person teaser. Can you pull and refresh? 
![Screenshot 2024-07-09 at 10 34 03 AM](https://github.com/SU-SWS/taiwan_hub_subtheme/assets/1187335/db60e0e2-d855-421e-913b-d1e1d8791ca9)